docs/general/passing_arbitrary_options.md

```diff
@@ -0,0 +1,45 @@
+## Passing Arbitrary Options To A Serializer
```

Comment:
Please add a line above this `[Back to Guides](../README.md)` and change the header level to h1 `# Passing`